packages/commonwealth/client/scripts/views/components/SublayoutHeader/useUserMenuItems.tsx

```diff
@@ -112,9 +143,40 @@ const useUserMenuItems = ({
     },
   );
 
+  const uniqueChainAddressOptions: PopoverMenuItem[] = uniqueChainAddresses.map(
+    (address) => {
+      const signed = true;
```

Comment:
this shouldn't be true => afair this should be based on session key

Reply:
actually this `uniqueChainAddressOptions` lists all addresses of users (that match active chain) and is only used if user is trying to switch address in a community where they are not a member + they want to buy stake in that community.

 Once they buy stake, they automatically join the community with that address and the actual value of "signed" (based on session key) is used.

Reply:
Ray had the great suggestion can we just name our variables better? pls?